### PR TITLE
Extend Analytics to include operating system ID and browser ID

### PR DESCRIPTION
**Pull request by stormybriggs1**
Issue #309 Extend Analytics to include operating system ID and browser ID.

This is an analytics specific concern that allows us to retain legacy browser and operating system references as we move to XDM.

### CODE REVIEW

**Comment by stormybriggs1** [comment]
I would like to get this merged in for the May 18th milestone. Thanks!

**Comment by stormybriggs1** [comment]
@kstreeter Would you mind looking at this short PR?

**Comment by kstreeter** [comment]
@stormybriggs1, what scheme is being used for these identifiers?  If they are something standard or commonly used I would suggest we add to the browserdetails or environment so they can be used elsewhere.  Are these from something like (for example) DeviceAtlas?

**Comment by stormybriggs1** [comment]
@cluby-adobe What are your thoughts on kstreeters comments?

**Comment by cluby-adobe** [comment]
@kstreeter, these are Analytics internal lookups.  They are not from a sharable standard.

File: extensions/adobe/experience/analytics/experienceevent.schema.json [hunk 1]
```diff
@@ -7212,6 +7212,18 @@
               "description": "Custom event 1000."
             }
           }
+        },
+        "https://ns.adobe.com/experience/analytics/environment/browserID": {
```
Comment (trieloff) [inline extensions/adobe/experience/analytics/experienceevent.schema.json]: Should be just `https://ns.adobe.com/experience/analytics/browserID`

File: extensions/adobe/experience/analytics/experienceevent.schema.json [hunk 2]
```diff
@@ -7212,6 +7212,18 @@
...
+          "title": "Browser ID",
+          "type": "integer",
+          "description":
+            "The Adobe Analytics Identifier for the browser used."
+        },
+        "https://ns.adobe.com/experience/analytics/environment/operatingSystemID": {
```
Comment (trieloff) [inline extensions/adobe/experience/analytics/experienceevent.schema.json]: Should be just `Should be just `https://ns.adobe.com/experience/analytics/operatingSystemID `

**Comment by lrosenthol** [comment]
If these values represent an *internal* concept, then they need to be in an extension schema and not a public one

**Comment by stormybriggs1** [comment]
@kstreeter @trieloff Please take another look. I have updated this PR as suggested by @cluby-adobe. Thanks.

**Pull request merged**